hw/dv/tools/dvsim/riviera.hjson

```diff
@@ -69,6 +69,13 @@
       build_opts: []
       run_opts: []
     }
+    {
+      name: riviera_gui_debug      
+      // TODO need to add options to setup Riviera gui debug environment and features (similary like in PR #24156 for Xelium)
```

Comment:
I think this would be better if we were pointing "internally". Can't you just say something like "Similar to how we do it for Xcelium: see xcelium.hjson." ?

Reply:
Yes. It can be described in this way.

Reply:
Probably just merge the changes together.